test/quantization/core/test_quantized_tensor.py

```diff
@@ -1602,6 +1602,14 @@ def test_decomposed_dequantize_per_channel(self):
         self.assertEqual(quantized_X.int_repr(), quantized_decomposed_X)
         self.assertEqual(dequantized_X, dequantized_decomposed_X)
 
+    def test_decomposed_choose_qparams_per_token_asymmetric_backward(self):
```

Comment:
is this used in QAT path? like choose_qparams -> fake_quant_op?

Reply:
yeah